docs/content/docs/concepts/summarization.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Summarization
+menuPosition: 10
```

Comment:
@tylerbutler Is this menuPosition needed? Also, I removed alias which was in the summarizer.md doc because there were no summarizer docs under those folders.